examples/openapi-ts-sample/src/App.tsx

```diff
@@ -9,14 +9,14 @@ import {
   Section,
 } from '@radix-ui/themes';
 
-// @ts-expect-error
-import { postFoo } from './client/sdk.gen';
+import { addPet } from './client/sdk.gen';
 
 function App() {
   const onClick = async () => {
-    postFoo({
+    addPet({
```

Comment:
Changed this file to use actual generated code, I was getting type errors how it was

Reply:
Sample is meant to be a playground for local development and testing. I thought I ignored it in ESLint?

Reply:
Maybe was an issue with my IDE, why not use actual code instead of adding broken imports and ignoring it with comments?

Reply:
I don't remember, I was probably testing something and didn't clean it up. It's similar to the sample snapshot script except it contains a whole app with UI. The idea is to be able to test drive the output since snapshots are never actually executed. Obviously should be way more polished!